cmd/juju/application/suspendrelation_test.go

```diff
@@ -37,7 +37,7 @@ func (s *SuspendRelationSuite) runSuspendRelation(c *gc.C, args ...string) error
 func (s *SuspendRelationSuite) TestSuspendRelationWrongNumberOfArguments(c *gc.C) {
 	// No arguments
 	err := s.runSuspendRelation(c)
-	c.Assert(err, gc.ErrorMatches, "no relation id specified")
+	c.Assert(err, gc.ErrorMatches, "no relation id\\(s\\) specified")
```

Comment:
Proper plural response please. No (s).

Reply:
this isn't a plural - it's an error message saying the user did not specify the required one (or more) relation ids. nonetheless, i've removed the brackets